src/sentry/static/sentry/app/components/group/externalIssuesList.tsx

```diff
@@ -116,19 +108,43 @@ class ExternalIssueList extends AsyncComponent<Props, State> {
     }
   }
 
+  async updateIntegrations(onSuccess = () => {}, onError = () => {}) {
+    try {
+      const {group} = this.props;
+      let integrations = await this.api.requestPromise(
+        `/groups/${group.id}/integrations/`
+      );
+      this.setState({integrations}, () => onSuccess());
+    } catch (error) {
+      onError();
```

Comment:
@NisanthanNanthakumar hmmm so if this fails, we just do nothing?

Reply:
@scefali if there is an error callback set, it will run. Ex: https://github.com/getsentry/sentry/pull/21657/files#diff-57efe1fe670bb8d26d9fe06e221cf3df919eea261009529b8113a5d917e0deebR76